pubmedRetraction.html

```diff
@@ -0,0 +1,18 @@
+meta(charset='utf-8')
+html
+  head
+    link(rel='stylesheet', href='PMR.css')
+  body
+    script(src='//d3js.org/d3.v3.min.js')
+    |  
+    form
+      select(name='graphs')
+        // placeholder text elements
+        option(value='contiYear') Continent / Year
+        |   
+        option(value='contYear') Country / Year
+        |   
+        option(value='jornYear') Journal / Year
```

Comment:
Use the full name for the variable names here. It is difficult to visually distinguish between `contiYear` and `contYear`.